packages/headless/src/controllers/commerce/core/facets/generator/headless-commerce-facet-generator.ts

```diff
@@ -88,10 +95,28 @@ export type CommerceSearchableFacetBuilder<
  *
  * The `FacetGenerator` options used internally.
  */
+/**
```

Comment:
1. There are two comments back-to-back here
2. Why do we need to document this type? It's internal to the commerce headless sub-package, and should not be exported.